corehq/apps/app_manager/static/app_manager/js/manage_releases_by_app_profile.js

```diff
@@ -0,0 +1,96 @@
+hqDefine('app_manager/js/manage_releases_by_app_profile', [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+    'hqwebapp/js/assert_properties',
+    'translations/js/app_translations',
+], function (
+    $,
+    ko,
+    _,
+    initialPageData,
+    assertProperties
+) {
+    'use strict';
+    $(function () {
+        var AppRelease = function (details) {
+            var self = {};
+            assertProperties.assertRequired(details, ['id', 'app_id', 'active', 'app_name', 'profile_name',
+                'version']);
+            self.id = details.id;
+            self.active = ko.observable(details.active);
+            self.status = details.active ? gettext('Active') : gettext('Inactive');
+            self.app_name = details.app_name;
+            self.version = details.version;
+            self.profile_name = details.profile_name;
+            self.domId = "restriction_" + self.id;
+            self.errorMessage = ko.observable();
+            self.domId = "restriction_" + self.id;
```

Comment:
Nitpick: this line is repeated two lines up. Also, what's `dom` short for?

Reply:
dom is DOM.

Reply:
https://github.com/dimagi/commcare-hq/pull/24692/commits/93daaf3fb93d0cf9f6699f1274e23edf94cd24dc